### ISSUE TITLE
pusher_beams_android:compileDebugKotlin

### ISSUE THREAD

**Issue by UzairJerry** [pusher/push-notifications-flutter#28]
e: /Users/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/pusher_beams_android-1.1.0/android/src/main/kotlin/com/pusher/pusher_beams/PusherBeamsPlugin.kt: (23, 1): Class 'PusherBeamsPlugin' is not abstract and does not implement abstract member public abstract fun onNewIntent(p0: Intent): Boolean defined in io.flutter.plugin.common.PluginRegistry.NewIntentListener
e: /Users/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/pusher_beams_android-1.1.0/android/src/main/kotlin/com/pusher/pusher_beams/PusherBeamsPlugin.kt: (45, 5): 'onNewIntent' overrides nothing

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':pusher_beams_android:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by fbenevides** [comment]
Hi @UzairJerry, thanks for reporting.

Our team is investigating this issue. Also, we're going to provide updates on it on #27 (which is duplicated).

**Comment by fbenevides** [comment]
Also, how are you declaring this dependency on your project?

Something like this?
```yml
dependencies:
  flutter:
    sdk: flutter
  pusher_beams: ^1.1.0
```

**Comment by Nick-Spruce-Goose** [comment]
> I'm facing the same issue. does anyone have a solution?

+1

**Comment by Thavananthan** [comment]
> I'm facing the same issue. does anyone have a solution?

+2

**Comment by joaomfariamartins** [comment]
> I'm facing the same issue. does anyone have a solution?

+3

**Comment by vincent-ikonk** [comment]
+4

**Comment by vincent-ikonk** [comment]
Same error : 
`/pusher_beams_android-1.1.0/android/src/main/kotlin/com/pusher/pusher_beams/PusherBeamsPlugin.kt: (23, 1): Class 'PusherBeamsPlugin' is not abstract and does not implement abstract member public abstract fun onNewIntent(p0: Intent): Boolean defined in io.flutter.plugin.common.PluginRegistry.NewIntentListener`

`dependencies:
  flutter:
    sdk: flutter
    pusher_beams: ^1.1.0`

**Comment by roldanunne** [comment]
Same issue here

**Comment by SolomonAbuh** [comment]
I'm facing the same challenge, I tried all versions of the pusher beam it didn't still work

**Comment by niglng** [comment]
Am facing same for the past 4-days now. Thinking it was my code or declaration. Below is my configuration

Flutter (Channel stable, 3.0.4, on Microsoft Windows [Version 10.0.19044.1826])
Android toolchain - develop for Android devices (Android SDK version 33.0.0)
Android Studio (version 2021.1)

environment:
   sdk: ">=2.17.1 <3.0.0"
   flutter: ">=3.0.1"

I had to drop it to
environment:
  sdk: ">=2.15.1 <3.0.0"

YET, the error continues.

**Comment by gtu-myowin** [comment]
Same issue happening to me. Here's the `flutter doctor -v` log

```
❯ flutter doctor -v
[✓] Flutter (Channel stable, 3.0.5, on macOS 12.5 21G72 darwin-x64, locale en-US)
    • Flutter version 3.0.5 at /Users/myowin/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f1875d570e (3 weeks ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/myowin/Library/Android/sdk
    • Platform android-32, build-tools 31.0.0
    • ANDROID_HOME = /Users/myowin/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/myowin/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.69.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.44.0

[✓] Connected device (2 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86 • Android 11 (API 30) (emulator)
    • macOS (desktop)                    • macos         • darwin-x64  • macOS 12.5 21G72 darwin-x64

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by thassio-vinicius** [comment]
FYI this issue is still going on. Temporary workaround for me is downgrading to `pusher_beams: ^1.0.2` and making sure the pub cache corrects the android version to `pusher_beams_android-1.0.2` (which, for me at least, doesn't present this problem) by running `flutter pub cache repair`.

**Comment by gtu-myowin** [comment]
@thassio-vinicius installing `pusher_beams: ^1.0.2` and `pusher_beams_android: 1.0.2` works. Thanks!

**Comment by TitanKing** [comment]
Also having the same issue

**Comment by fbenevides** [comment]
Could you please check the latest version? We've released 1.1.1

**Comment by benw-pusher** [comment]
This should have been resolved in version 1.1.1. If you continue to experience issues please reach out via https://support.pusher.com/hc/en-us/requests/new
